### DISCUSSION TITLE
way to create adminclaims in world disable grief prevention

### DISCUSSION THREAD

**Discussion by lucasryw** [GriefPrevention/GriefPrevention#1063] [Q&A] [answered]
hi! my config file set mode disabled for nether and the end, but I want admins to be able to create claims there, to protect some things. is this possible? thank you

**Comment by bigpresh** [comment, marked as answer]
With it disabled for a given world, *no* checks will be done there.

You could enable it for them, but remove the `griefprevention.claim` permission (and possibly `griefprevention.createclaims`) from players for that world using any supported permissions plugin which lets you set permissions per-world.

That way, players couldn't create claims, as it sounds like that's what you want, but your admins could still have `griefprevention.adminclaims`, and GP would protect those claims.

**Comment by lucasryw** [comment]
nice, thanks man!
